addons/im_livechat/static/src/core/common/message_model_patch.js

```diff
@@ -12,7 +12,9 @@ const messagePatch = {
     canReplyTo(thread) {
         return (
             super.canReplyTo(thread) &&
-            (thread?.channel_type !== "livechat" || !thread.composerDisabled)
+            (thread?.channel_type !== "livechat" ||
+                !thread.composerDisabled ||
+                !thread.composerHidden)
```

Comment:
```suggestion
            if (this.thread?.channel_type === "livechat") {
                return !thread.composerDisabled && !thread.composerHidden;
            }
            return super.canReplyTo(thread);
```

I feel like we reached the point where it would be easier to read with a dedicated block.